packages/integrations/mdx/src/remark-images-to-component.ts

```diff
@@ -89,6 +95,55 @@ export function remarkImageToComponent() {
 					});
 				}
 
+				if (node.data && node.data.hProperties) {
+					const { widths, densities, ...prop } = node.data.hProperties as imgArgs;
```

Comment:
Is there a way we could make this more generic? `widths` and `densities` are attributes our image services supports, but other image services might have more (or less) attributes. Maybe doing some sort of switch with a typeof and automatically making them in the first format would be a better approach

Reply:
That is much better you're right